uperf/uperf_run

```diff
@@ -451,8 +450,12 @@ execute_test()
 				ethtool -S ${nets} > ${uperf_results}/net_stats_${nets}_iter_${test_iteration}.before
 			done
 		fi
-		/usr/local/bin/uperf -m xml${net_count} >> $results_file_worker &
-		pids[${net_count}]=$!
+		for client in $client_ip_list; do
+			scp xml${net_count} root@$client:/root/xml${net_count} > /dev/null
```

Comment:
For future-proofing, we need to see if /root will be writeable for bootc-based images.  It's unclear from the Fedora documentation if the root user will still use /root (it references /var/roothome) for its writeable area.